### ISSUE TITLE
If visually, link and text appear similar, Can we map it in any WCAG guideline as failure?

### ISSUE THREAD

**Issue by rahulpundir019** [w3c/wcag#4600]
In the image below, we have a breadcrumb where the first item is a link in grey color(underline appears on mouse hover), and the second item is just normal text, which also looks the same as a link

<img width="591" height="188" alt="Image" src="https://github.com/user-attachments/assets/a5e94d12-b2e5-433c-a592-b36ef3ef0c09" />

Raising this bug to clarify if we can map this to any WCAG guideline.
